src/main/java/ensa/ebanking/accountservice/DAO/ClientProfileDAO.java

```diff
@@ -14,7 +14,5 @@ public interface ClientProfileDAO extends JpaRepository<ClientProfile, Long> {
 
     List<ClientProfile> findClientProfileByAccountStatus(AccountStatus inactive);
 
-    ClientProfile findClientProfileById(long id);
```

Comment:
Why is this removed?

Reply:
Worked with one already there in spring data that returns Optional<ClientProfile> which returns a ClientProfile if there's one in the db and a null if there isn't.

I haven't tested the app with "ClientProfile findClientProfileById(long id);", but the other one works and this one wasn't used anywhere else so it being removed doesn't affect anything.

Reply:
The request comes with an id not with the entire ClientProfile. I see that you changed it. Does bader know about this ?

Reply:
No, I didn't tell him about this. The method parameter was a String Json and I thought it had a ClientProfile in the body as a json. I can turn this back to just an id in the body, or I can tell Bader to send ClientProfile. Whichever you choose.

Reply:
Do whichever you want, either talk with him and leave it as clientprofile or revert it to id.